### PR TITLE
import: add '--file' option

### LINKED ISSUE #4093: Support `dvc import --file`
Currently, `--file` is supported on `dvc add` and `dvc import-url`, but not on `dvc import`.  This functionality would be useful for keeping `.dvc` files in a special place.

### PR DESCRIPTION
**Pull request by DiPaolo**
Fixes #4093

* [x] ❗ I have followed the [Contributing to DVC](https://dvc.org/doc/user-guide/contributing/core) checklist.

* [x] 📖 If this PR requires [documentation](https://dvc.org/doc) updates, I have created a separate PR (or issue, at least) in [dvc.org](https://github.com/iterative/dvc.org) and linked it here.

https://github.com/iterative/dvc.org/pull/1816

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

File: dvc/command/imp.py [hunk 1]
```diff
@@ -59,4 +60,9 @@ def add_parser(subparsers, parent_parser):
         help="Git revision (e.g. SHA, branch, tag)",
         metavar="<commit>",
     )
+    import_parser.add_argument(
+        "--file",
+        help="Specify name of the DVC-file this command will import.",
```
Comment (efiop) [inline dvc/command/imp.py:65]:
```suggestion
        help="Specify name of the DVC-file this command will generate.",
```

**Review by efiop — APPROVED** [review]
Looks great! Thank you @DiPaolo ! :pray:

**Comment by efiop** [comment]
@DiPaolo Looks like we need to update the docs for this one. Could you please create an issue in https://github.com/iterative/dvc.org or adjust https://dvc.org/doc/command-reference/import ?

**Comment by DiPaolo** [comment]
@efiop done in https://github.com/iterative/dvc.org/pull/1816. I've just copy-pasted the description form similar import-url (https://dvc.org/doc/command-reference/import-url) command. That looks correct to me. But it's better to double check.

**Comment by efiop** [comment]
@DiPaolo Thank you! :pray:

**Pull request merged**